cmd/manager/main.go

```diff
@@ -90,8 +90,10 @@ func main() {
 		"Address for hosting metrics",
 	)
 
-	klog.InitFlags(nil)
 	flag.Set("logtostderr", "true")
```

Comment:
i think this line is causing the issue, not sure if it needs to happen after the textlogger init or if we need to add an option.